integration/js/utils/AppTestStep.js

```diff
@@ -18,7 +18,7 @@ class AppTestStep extends TestStep {
       this.driver = sessionInfo.driver;
     }
 
-    this.namespaceInfix = kiteBaseTest.payload?.namespaceInfix || '';
+    this.namespaceInfix = kiteBaseTest.payload && kiteBaseTest.payload.namespaceInfix || '';
```

Comment:
This is kinda hard to read, can we just do a normal if else?

Reply:
Remove the guard, which is not very necessary. Cause we need `url` in `payload`, which means `payload` should not be `undefined` anyway. And the reset of code don't have this kind of guard.